src/styles/sidebar/components/tags-input.scss

```diff
@@ -10,16 +10,16 @@ tags-input {
   }
 
   .tags {
+    display: flex;
+    flex-wrap: wrap;
```

Comment:
👍 for migrating floats & clearfixes etc. to modern layout systems (flexbox, grid)